src/engine/Engine.h

```diff
@@ -389,25 +389,25 @@ class Engine {
   static void createOptionalResult(const typename A::value_type* a,
                                    const typename B::value_type* b,
                                    size_t sizeA,
-                                   int jcls_a, int jcls_b,
-                                   const std::vector<size_t>& jclAToB,
+                                   int joinColumBitmap_a, int joinColumBitmap_b,
```

Comment:
It's "column" so use either "..Col" or "..Column"

Reply:
fixed